crates/flox-rust-sdk/src/models/search.rs

```diff
@@ -223,6 +225,24 @@ pub enum Subtree {
 #[derive(Debug, Clone, Serialize)]
 pub struct SearchResults {
     pub results: Vec<SearchResult>,
+    pub count: Option<u64>,
+}
+
+/// The types of JSON records that `pkgdb` can emit during a search
+#[derive(Debug, PartialEq, Deserialize)]
+#[serde(untagged)]
+pub enum Record {
+    /// A record containing the total number of search results regardless
+    /// of how many are displayed to the user
+    #[serde(untagged)]
+    #[serde(rename_all = "kebab-case")]
+    ResultCount { result_count: u64 },
+    /// A single search result
+    #[serde(untagged)]
+    SearchResult(SearchResult),
+    /// An error
+    #[serde(untagged)]
+    Error(PkgDbError),
```

Comment:
suggestion(armadilloable): remove inner `untagged` annotations

`untagged` on the enum itself should apply to all variants already

```suggestion
    #[serde(rename_all = "kebab-case")]
    ResultCount { result_count: u64 },
    /// A single search result
    SearchResult(SearchResult),
    /// An error
    Error(PkgDbError),
```

Reply:
Fixed